### ISSUE TITLE
Running on Android

### ISSUE THREAD

**Issue by htzmedia** [zzh8829/yolov3-tf2#65]
Hello, I am trying to run my custom trained model on android.
Can anyone give me some help? I have converted the Keras model to tflite. 
But I don't know how to run it on Android, is there any basic (easy to understand) example Apps?
I tried to use the Tensorflow Object Detection Example App, but without luck.

**Comment by PelayoChoya** [comment]
Hi,

I am also facing this problem. Did you figure out any solution?

**Comment by htzmedia** [comment]
Hi, in another issue I described how I converted the model to tflite. On my PC it throws Error that TFLITE doesn't support BatchNormalization but on my Notebook and the same code everything worked just fine. 
For Running it on Android I customized the Tensorflow Object Detction Example App.

If you have any Questions feel free to ask
